Library/Formula/collectd.rb

```diff
@@ -22,17 +22,53 @@ class Collectd < Formula
     depends_on "autoconf" => :build
   end
 
-  option "with-java", "Enable Java support"
-  option "with-protobuf-c", "Enable write_riemann via protobuf-c support"
+  option "with-amqp", "Enable AMQP plugin"
+  option "with-curl_json", "Enable Curl JSON plugin"
+  option "with-dbi", "Enable DBI plugin"
   option "with-debug", "Enable debug support"
+  option "with-java", "Enable Java support"
+  option "with-gmond", "Enable Ganglia gmond plugin"
+  option "with-libvirt", "Enable Libvirt plugin"
+  option "with-memcachec", "Enable memcachec plugin"
+  option "with-modbus", "Enable Modbus plugin"
+  option "with-mysql", "Enable MySQL plugin"
+  option "with-notify_desktop", "Enable Desktop notifications plugin"
+  option "with-nut", "Enable Nut plugin"
+  option "with-onewire", "Enable OneWire plugin"
+  option "with-pinba", "Enable Pinba plugin"
+  option "with-ping", "Enable Ping plugin"
+  option "with-postgresql", "Enable PostgreSQL plugin"
+  option "with-redis", "Enable Redis plugin"
+  option "with-varnish", "Enable Varnish plugin"
+  option "with-write_kafka", "Enable write_kafka plugin"
+  option "with-write_redis", "Enable write_redis plugin"
+  option "with-write_riemann", "Enable write_riemann plugin"
 
   deprecated_option "java" => "with-java"
   deprecated_option "debug" => "with-debug"
 
-  depends_on "pkg-config" => :build
-  depends_on "protobuf-c" => :optional
+  depends_on "yajl" if build.with? "curl_json"
```

Comment:
These options should be instead named `depends_on "yajl" => :optional` and `option "with-yajl", "Enable Curl JSON plugin"`